### PR TITLE
Features/redesign proposal details page

### PR DESCRIPTION
**Pull request by mmrrnn**

### CODE REVIEW

**Comment by mmrrnn** [comment]
Screenshot from 2022-05-18 15-08-44

File: src/pages/ProposalDetails/VotesBlock/AgainstVotes/AgainstVotes.component.tsx [hunk 1]
```diff
@@ -4,6 +4,15 @@ import { VoteAgainstButton } from '../VotesBlock.voteButtons';
 import { VoteActionBlock } from '../VotesBlock.voteAction';
```
Comment (mbalazy) [inline src/pages/ProposalDetails/VotesBlock/AgainstVotes/AgainstVotes.component.tsx:4]: two different components (AgainstVotesComponent and ForVotesComponent) are ok, but they are very similar. 
I will create some base component and pass props into it, with `iconSrc`, `type` etc.

File: src/pages/ProposalDetails/VotesBlock/VotesBlock.voteButtons.tsx [hunk 2]
```diff
@@ -1,6 +1,9 @@
+import { Typography } from '@mui/material';
```
Comment (mbalazy) [inline src/pages/ProposalDetails/VotesBlock/VotesBlock.voteButtons.tsx:1]: imo same case as VotesComponent

**Review by mbalazy — DISMISSED** [review]

File: src/pages/ProposalDetails/VotesBlock/AgainstVotes/AgainstVotes.container.tsx [hunk 3]
```diff
@@ -7,8 +7,7 @@ import {
   againstVoteStatusSelector,
```
[the remark this answers is not in the document]
Comment (mmrrnn) [reply src/pages/ProposalDetails/VotesBlock/AgainstVotes/AgainstVotes.container.tsx:7]: done

**Review by meho99 — APPROVED** [review]

**Pull request merged**